engine/client/s_main.c

```diff
@@ -1927,6 +1927,7 @@ qboolean S_Init( void )
 	Cmd_AddCommand( "spk", S_SayReliable_f, "reliable play a specified sententce" );
 	Cmd_AddCommand( "speak", S_Say_f, "playing a specified sententce" );
 
+	dma.backendName = NULL;
```

Comment:
Do you really need to set it to NULL beforehand?

Reply:
I think better to do it, that's will protect if dma.backendName won't set in some of audio backends, for example

Reply:
@SNMetamorph we better not have such backends at all. :)

Also, globals are zero initialized anyway by standard.

Reply:
Ok

Reply:
May be set "None" as backend by default?

Reply:
Yes